hooks/utils.py

```diff
@@ -652,18 +605,14 @@ def fetch_gui_release(origin, version):
     return download_release(url, filename)
 
 
-def setup_gui(release_tarball):
+def setup_gui(release_tarball_path):
```

Comment:
`<bikeshed>`
Should we use 'python_package_path' or something? It is a tarball, but it would make it clearer that we're not using the old tarball stuff anymore.
`</bikeshed>`